src/app/components/Search/Search.jsx

```diff
@@ -163,14 +163,6 @@ class Search extends React.Component {
           />
           <SearchButton onClick={this.submitSearchRequest} />
         </div>
-        <span
```

Comment:
This can be left here since it does make sense to have this error message with the component. The other error message for the date can be left alone.

Reply:
So we have two places with two different errors, is this what you suggest? So in <SearchResults> we have to decide which error goes to <Search>, and which remains.

Reply:
I feel like all the error visually in the same place is better, but we can definitely pass the related error to <Search> as an reference or future use.

Reply:
1) Yes, two different places, for now. One would be inside the Search component which renders the 'no keyword' error, and the other would be the date error which we haven't *fully* agreed yet. But for now, outside in the `SearchResultsPage.jsx` component it's fine.

2) I agree and I did say that visually in the same location is good. I just do not agree about introducing a 'date error' variable and code into a component that doesn't worry about date and only handles searches. It just bloats the component and if we have to look back after a month to update the date error message, it'd be confusing to look for it in the `Search.jsx` component

Reply:
Ok thanks for the clarifying.